tests/bsim/bluetooth/mesh/tests_scripts/bridge/brg_simple_group.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+# Copyright 2024 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
+
+# This test checks basic functionality of the Subnet Bridge with group addressing. It checks the
```

Comment:
Additional thanks for good test scenario description. It is nightmare to support complex tests without good scenario description.